pulsar/client/interface.py

```diff
@@ -85,10 +83,13 @@ def __init__(self, destination_params, transport):
         self.transport = transport
         remote_host = destination_params.get("url")
         assert remote_host is not None, "Failed to determine url for Pulsar client."
-        if not remote_host.endswith("/"):
-            remote_host = "%s/" % remote_host
         if not remote_host.startswith("http"):
             remote_host = "http://%s" % remote_host
+        manager = destination_params.get("manager", None)
+        if manager:
+            remote_host = urljoin(remote_host, "managers/%s" % manager)
+        if not remote_host.endswith("/"):
```

Comment:
For backward compatibility can you check if the URL contains ``managers/`` already and warn the user the supplied manager will be ignored?